style.css

```diff
@@ -477,6 +477,109 @@ textarea {
   margin-left: 130px;
 }
 
+/* mobile popup------------------------------------------- */
+.projdesk {
+  display: none;
+}
+
+.modal-mobile {
+  overflow-x: hidden;
+  position: fixed;
+  top: 70px;
+  left: 15px;
+  background-color: white;
+  width: 90%;
+  display: none;
+  border: 1px solid #ebecf0;
+  box-shadow: 0 48px 48px rgba(0, 0, 0, 0.763);
+}
+
+.modal-mob {
+  margin-left: 10px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+}
```

Comment:
- [ ] [REQUIRED] Please make `Modal` all elements for the `Mobile version` look like [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) because I don't see them in your mobile version so please fix the style errors!

![image](https://user-images.githubusercontent.com/63915024/188954708-41f10605-f154-4730-8302-78012a11262f.png)